tee-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -168,6 +168,35 @@ impl TrustedCall {
 		}
 	}
 
+	pub fn req_hash(&self) -> Option<&H256> {
```

Comment:
Can be renamed to `call_id`. I think it's worth to abstract it a little bit and make `stf` less req/rpc aware.

Reply:
I'll use `identifier`